components/third-party/kube-prometheus-stack/chart/helmRelease.yaml

```diff
@@ -23,7 +23,9 @@ spec:
     grafana:
       enabled: false
 
-    prometheusOperator: #
+    prometheusOperator:
+      podLabels:
+        grafana.radix.equinor.com/monitor-health: "true"
```

Comment:
Do we also need to specify podLabels for node-exporter, ref https://artifacthub.io/packages/helm/prometheus-community/kube-prometheus-stack?modal=values&path=prometheus-node-exporter.podLabels?